docs/posts/overview-of-ballerina-0.980.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Overview of Ballerina 0.980
+author: Kishanthan Thangarajah
+date: 26 July 2018
+status: Published
+abstract: Overview of new features and improvements added in ballerina 0.980.0 release
+socialmediaimage: ballet-sneaker-dress-ballet-dancer-163379.jpeg
+---
+
+# Improvements and New Features
+Ballerina 0.980.0 release includes some type system improvements and new features additions. This release also focused on stabilizing the platform with lot of bug fixes and improvements. Following are the new changes and improvements added with the 0.980.0 release.
```

Comment:
Change to 'Ballerina 0.980.0 release included some type system improvements and new features. This release also focused on stabilizing the platform with lot of bug fixes and improvements.'